README.md

```diff
@@ -285,7 +108,3 @@ I would like to thank...
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly fix this link, at the moment, it goes to a 404, page not found. This is necessary because it is upon this link your repo takes legal permission to exist.